Library/.rubocop.yml

```diff
@@ -8,7 +8,15 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
There are classes in files
`compat/formula_specialties.rb`
`test/exceptions_spec.rb`
`test/support/fixtures/testball.rb`
`test/support/fixtures/testball_bottle.rb`
 inheriting from `Formula` in these files and are returning cop violations as they don't have formula `desc`, causing the build to fail. If `desc` can be added in the above files, then we don't have to repeat `Include` in `.rubocop.yml`